pkg/procs/info.go

```diff
@@ -0,0 +1,160 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * Copyright 2020 Red Hat, Inc.
+ */
+
+package procs
+
+import (
+	"bufio"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"strconv"
+	"strings"
+
+	"k8s.io/kubernetes/pkg/kubelet/cm/cpuset"
+)
+
+type TIDInfo struct {
+	Tid      int
+	Name     string
+	Affinity []int
+}
+
+type PIDInfo struct {
+	Pid  int
+	Name string
+	TIDs map[int]TIDInfo
+}
+
+func All(procfsRoot string) (map[int]PIDInfo, error) {
+	infos := make(map[int]PIDInfo)
+	pidEntries, err := ioutil.ReadDir(procfsRoot)
+	if err != nil {
+		return infos, err
+	}
+
+	for _, pidEntry := range pidEntries {
+		if !pidEntry.IsDir() {
+			continue
+		}
+		pid, err := strconv.Atoi(pidEntry.Name())
+		if err != nil {
+			// doesn't look like a pid
+			continue
+		}
+
+		pidInfo, err := FromPID(procfsRoot, pid)
+		if err != nil {
+			// TODO log
```

Comment:
Yep, maybe a github issue in the meantime?

Reply:
same comment as above, do we need to merge this PR to have issues, don't we?